### PR TITLE
Quick Installer should specify which config file to edit.

### PR DESCRIPTION
**Pull request by jkaurredhat**
Fixes bugzilla : https://bugzilla.redhat.com/show_bug.cgi?id=1460353

Signed-off-by: jkaurredhat <jkaur@redhat.com>

### CODE REVIEW

File: utils/src/ooinstall/cli_installer.py [hunk 1]
```diff
@@ -797,7 +797,7 @@ def run_config_playbook(oo_cfg, hosts_to_run_on, unattended, verbose, gen_invent
 
     click.echo('Ready to run installation process.')
     message = """
-If changes are needed please edit the config file above and re-run.
+If changes are needed please edit the installer.cfg.yml config file above and re-run.
```
Comment (tbielawa) [inline utils/src/ooinstall/cli_installer.py]: Looks good to me!

**Comment by tbielawa** [comment]
aos-ci-test

**Comment by tbielawa** [comment]
[test]

**Comment by jkaurredhat** [comment]
Thanks for the review. If required should this be backported to previous versions also ?

**Pull request merged**